source/exploring_catalog_and_datasets/01_looking_up_the_catalog/catalog.rst

```diff
@@ -1,79 +1,179 @@
 
-Full text search
-~~~~~~~~~~~~~~~~
+.. admonition:: Note
+   :class: note
+
+   By default, datasets for which the geographic area metadata has not been set cannot be found when doing a geographic search with the mini map.
+   Furthermore, the mini map is only available and displayed on the catalog if at least one of the datasets has this geographic area metadata set.
 
-The text box filters the list thanks to a full text search across all the datasets. It means it will show any dataset that contains the searched text in any of its metadata.
+To make a geographic search:
 
-.. image:: images/filters_search.png
-   :alt: Search box
+1. Click on the little rectangle button on the top left of the mini map, which is a drawing tool.
+2. Draw a rectangle on the chosen area of the mini map. The catalog directly updates to only display datasets matching the geographic filter applied.
 
-Type the searched text in the box and click on the magnifying glass pictogram to filter the list via a full text search.
+To cancel a selection, and draw on another area:
 
-Geographic search
-~~~~~~~~~~~~~~~~~
+1. Click on the rectangle button again. The previous shape diseappears.
+2. Draw a rectangle on the newly chosen area of the mini map. The catalog directly updates to only display datasets matching the geographic filter newly applied.
 
-Datasets are often closely related to a geographic area, be it a country, a city, a district, etc.
+Using the selectable filters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The map filters the list by location. It will only show datasets that contain geographic data (geopoints or geoshapes mainly) inside the selected rectangle on the map.
+Under the mini map is a list of selectable filters sorted by categories based on the datasets metadata. These filters should be used to filter the datasets by view (visualizations such as Analyze, Map, Calendar etc.), date, publisher, keyword, theme, language, and any other metadata available as filter category. When searching with the selectable filters, the platform will look for a match between the selected values from the filter categories and the values set in the metadata of the available datasets. The catalog will be filtered to only display those matching datasets.
 
-.. localizedfigure:: images/geographic_search__explore.png
+To use the selectable filters:
 
-   The mini-map used for filtering on the catalog page.
+1. Choose a category of filters.
+2. Under the name of the category of filters are listed values (which are the metadata set for the datasets). Click one of the values. The other values disappear and the catalog directly updates to only display datasets matching the newly applied filter.
```

Comment:
"Under the category name of filters" ?

Reply:
I think I will keep it as it is because it seems easier to understand (in my opinion).